.asf.yaml

```diff
@@ -52,3 +52,24 @@ github:
     squash: true
     merge: false
     rebase: false
+
+  protected_branches:
+    master:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # contexts are the names of checks that must pass
+        contexts:
+          - Docker/build
+          - Frontend/build
+          - PR Lint/check
+          - E2E/Cypress
+          - lint
+          - test-mysql
+          - test-postgres
+          - test-sqlite
+
+      required_pull_request_reviews:
+        dismiss_stale_reviews: true
```

Comment:
Do we want this? It might cause some pain, say if a PR is approved but a rebase is needed before merging.

Reply:
good call. updated.